bricksrc/equipment.py

```diff
@@ -742,19 +880,41 @@
         },
         "parents": [BRICK.Water_Heater],
     },
-    "Metering_Device": {
-        "tags": [TAG.Refrigerant, TAG.Gas, TAG.Liquid, TAG.Metering, TAG.Expansion, TAG.Valve],
+    "Throttling_Device": {
```

Comment:
Though I do prefer to use Metering Device, I have renamed "Metering Device" to "Throttling Device" to avoid confusion with Brick's [Meter ](https://brickschema.org/ontology/1.3/classes/Meter/) class. Throttling device can refer to any component that restricts or regulates the flow of a fluid. It's not specific to refrigeration systems and can apply to various types of mechanical and fluid systems. Open to feedback on this, as I would prefer to use Metering Device if we didn't have a naming conflict.

Reply:
I'm not familiar with the area, but this substitution seems fine for me. We could also go "Metering Device" and then add shackle rules to either Meter class or this class that detect when it's being used incorrectly.

Reply:
I'm not either, but in general I always recommend/want terms to be rooted in a standard (ASHRAE, ISO, etc.). Does `Throttling Device` come from somewhere? 

I see `Throttling Valve` in ASHRAE terminology (although it's not a standard nor are their standards required to use these). 

[Throttling Valve](https://terminology.ashrae.org/?term=throttling)
> valve used to restrict (throttle) the flow of fluid.

Reply:
@gtfierro Ok, great to hear. In that case, I would prefer to call it "Metering Device". @MatthewSteen I was careful not to use the name "valve", due to the various metering devices used in practice, though I do agree that Throttling Device is a bit broad. If you're curious to learn more about metering devices, this is an informative article:
https://www.skillcatapp.com/post/a-complete-guide-to-metering-devices-fixed-modulating.

With the addition of metering device, Brick now has the ability to represent a complete refrigeration cycle using the following classes:
- Compressor
- Reversing Valve (if Heat Pump)
- Metering Device
- Evaporator Coil
- Condenser Coil

![refrigerant-flow copy](https://github.com/BrickSchema/Brick/assets/33592545/7bf10228-618d-4dda-b6da-48f855f29874)

Reply:
I strongly suggest basing the terminology on existing definitions/terms that are as close to canonical as possible so we're not "inventing" our own or basing them on a website. There are a lot of interchangeable terms used in industry but I think Brick should be using "authoritative" sources when it comes to its class names whenever possible. 

I may have missed it, but what's the problem with using `Throttling_Valve` per ASHRAE terms instead of `Metering_Device`, which seems a bit general for this application (VRF). Seems like the following hierarchy might work...

Valve
- Reversing Valve
- Throttling Valve

Reply:
Agreed that reusing ASHRAE terms is ideal. Between the two, I would prefer "Refrigerant Metering Device".

Throttling_Valve is also a good alternative. However, Throttling Valve would only be able to represent [modulating metering devices](https://www.skillcatapp.com/post/a-complete-guide-to-metering-devices-fixed-modulating#viewer-3cgvk) and would not be able to represent [fixed metering devices](https://www.skillcatapp.com/post/a-complete-guide-to-metering-devices-fixed-modulating#viewer-vdi1).

Reply:
I think we should be basing Brick's class names on an authoritative/canonical source (such as AHRI, ANSI, ASHRAE, ISO, IEE, etc.), *not* https://www.skillcatapp.com/.

Reply:
100% agreed, I shared the article with you to explain the differences between fixed and modulating metering devices and how throttling valve does not encompass all metering devices.

I have renamed the class to refrigerant metering device per ASHRAE's definition:
https://terminology.ashrae.org/?entry=refrigerant%20metering%20device